source/extensions/filters/network/thrift_proxy/stats.h

```diff
@@ -29,7 +29,7 @@ namespace ThriftProxy {
   COUNTER(response_reply)                                                                          \
   COUNTER(response_success)                                                                        \
   GAUGE(request_active, Accumulate)                                                                \
-  HISTOGRAM(request_time_ms)
+  HISTOGRAM(request_time, Milliseconds)
```

Comment:
ditto for messaging out the update.

Reply:
Consolidating the rename discussion in the OP.